mockgcp/mocknetapp/backupvault.go

```diff
@@ -75,10 +75,7 @@ func (s *backupVaultsService) CreateBackupVault(ctx context.Context, req *pb.Cre
 		return nil, err
 	}
 
-	metadata := &pb.OperationMetadata{}
-	return s.operations.StartLRO(ctx, req.GetParent(), metadata, func() (proto.Message, error) {
-		return obj, nil
-	})
+	return s.operations.DoneLRO(ctx, req.GetParent(), &pb.OperationMetadata{}, obj)
```

Comment:
Not sure why this is changed to `DoneLRO`? Isn't `StartLRO` a better mock for the real GCP behavior?

Reply:
I think I was seeing issues running the mock when using StartLRO as opposed to just returning a finished operation.

Reply:
Not sure if it is related, but the service.go is missing registering the operation HTTP path. e.g.
https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/6ca97c177142ece82bc5850989a5313dab59f5ca/mockgcp/mockbackupdr/service.go#L67C3-L67C76